src/scss/skin-super-modern/_mixins.scss

```diff
@@ -137,10 +137,10 @@
   &:focus {
     box-shadow: $focus-element-box-shadow;
     outline: none;
-  }
 
-  &:focus:not(.#{$prefix}-focus-visible) {
-    box-shadow: none;
-    outline: none;
+    &:not(.#{$prefix}-focus-visible) {
```

Comment:
Why did you remove this `focus`? 🤔

Reply:
It is still there. The `:not` just nested in the `&:focus`. Otherwise the linter would throw a warning.

```
&:focus {
    box-shadow: $focus-element-box-shadow;
    outline: none;

    &:not(.#{$prefix}-focus-visible) {
      box-shadow: none;
      outline: none;
    }
  }

```

Reply:
Sorry missed that 🤦